atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -160,7 +174,7 @@ public void delete(TableReference tableRef, Multimap<Cell, Long> keys) {
     @Override
     public void deleteRange(TableReference tableRef, RangeRequest range) {
         maybeLog(() -> delegate.deleteRange(tableRef, range),
-                logTimeAndTableRange("deleteRange", tableRef, range));
+                logTimeAndTableRangeOfIterator("deleteRange", tableRef, range));
```

Comment:
why did this one change?

Reply:
Refactored and didn't notice this had changed. Reverted.